File: codesignal/arcade/2.core/9.well of integration/6.addBorder.cpp
// Given a rectangular matrix of characters, add a border of asterisks(*) to it.

#include <iostream>
#include <vector>

class Solution {
public:
  std::vector<std::string> addBorder(std::vector<std::string> picture) {
    std::vector<std::string> result;
    std::string border(picture[0].size() + 2, '*');
    result.push_back(border);
    for (auto& s : picture) {
      result.push_back("*" + s + "*");
    }
    result.push_back(border);
    return result;
  }
};

File: codesignal/arcade/2.core/9.well of integration/6.addBorder.py
# Given a rectangular matrix of characters, add a border of asterisks(*) to it.

class Solution:
      def addBorder(self, picture):
          for i in range(len(picture)):
              picture[i] = "*" + picture[i] + "*"
          picture.insert(0, "*" * len(picture[0]))
          picture.append(picture[0])
          return picture